src/lib/gemini.ts

```
@@ -21,4 +26,66 @@ export async function generateGeminiSummary(notes: ShortTermNote[]): Promise<str
+    const response = await result.response;
+    const tagsText = response.text().trim();
+    
+    // Parse the response and clean up tags
+    const tags = tagsText
+      .split(',')
+      .map(tag => tag.trim().toLowerCase())
+      .filter(tag => tag.length > 0 && tag.length <= 20) // Max 20 chars per tag
+      .slice(0, 5); // Max 5 tags
+    
+    return tags;
+  } catch (e: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)